operations/Makefile

```
@@ -1,16 +1,24 @@
 SHELL := bash
 .SHELLFLAGS := -eu -o pipefail -c
 
-TF_ENV = "dev"
-TF_ENV_MODULE = ""
+TF_ENV = "dev" #VAR Your target environment (e.g. dev, test, staging, prod)
+TF_STAGE = ""  # not surfaced
 
 # HELP
 # This will output the help for each task
 # thanks to https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
-.PHONY: help
+.PHONY: help all tf-cli tf-01-network tf-02-config tf-03-persistent tf-04-app
```

Remove the `all` target. See comment below.

```suggestion
.PHONY: help tf-cli tf-01-network tf-02-config tf-03-persistent tf-04-app
```

---

see 46206085f13adc14a3f7b7e46a43415e3c6869b3